### DISCUSSION TITLE
node_modules/discord-player/dist/index.d.ts:2:41 - error TS2724: '"discord.js"' has no exported member named 'StageChannel'. Did you mean 'StoreChannel'?

### DISCUSSION THREAD

**Discussion by k3lux** [Androz2091/discord-player#761] [General]
**Describe the bug**
the StageChannel class that discord-player is importing does work.

**Please complete the following information:**
 - Node Version: latest
 - Discord Player Version: latest
 - Discord.js Version: latest

**The error**
node_modules/discord-player/dist/index.d.ts:2:41 - error TS2724: '"discord.js"' has no exported member named 'StageChannel'. Did you mean 'StoreChannel'?

**Comment by Samiul30** [comment]
Why this error was triggered? What was the command? Or work?

**Comment by twlite** [comment]
Something is not right with your discord.js, try re-installing
